src/components/TeamMemberTasks/ReviewButton.jsx

```diff
@@ -51,7 +52,7 @@ const ReviewButton = ({
         Submit for Review
       </Button>;
      } else if (reviewStatus == "Submitted")  {
-      if (myRole == "Owner" ||myRole == "Administrator" || myRole == "Mentor" || myRole == "Manager") {
+      if (myRole == "Owner" ||myRole == "Administrator" || myRole == "Mentor" || myRole == "Manager" || userPermission) {
```

Comment:
```suggestion
      const canPutReviewStatus = props.hasPermission('putReviewStatus')
      if (canPutReviewStatus) {
```
This is unrelated to this PR and should be either added to the description/testing instructions or removed/remade in its own PR.

Wherever you choose to add the `putReviewStatus` permission, it should replace any role logic with standardized permission logic.
Import and connect `hasPermission()` when you're implementing this.